mod.yaml

```diff
@@ -19,6 +19,7 @@ Packages:
 	$yupgi_alert: yupgi_alert
 	$ra: ra
 	$cnc: cnc
+	$as: as
```

Comment:
I have no idea what're you talking about. You shouldn't expect a released mod suddenly depending on an unreleased one entirely. And no way I would adjust to this by releasing an AS build.

The modtemplate shouldn't say anything about mods depending on each other, that feature is getting removed!